kuma/wiki/models.py

```diff
@@ -1479,18 +1471,9 @@ def zone_stack(self):
 
     def get_share_url(self):
```

Comment:
I was keeping it a method in case we ever need it later and it is one place to update. But I agree, I like your method name better.